src/tasks/package.js

```diff
@@ -25,12 +25,12 @@ const packages = [
   'pre-commit',
 ];
 
-module.exports = () => {
+module.exports = (config) => {
   json('package.json')
     .merge({
       engines: {
         // Some versions are skipped because of known issues, see https://github.com/webpack-contrib/organization/issues/7
-        node: '>=4.3.0 <5.0.0 || >=5.10',
+        node: `>=${config.minNode} <5.0.0 || >=5.10`,
```

Comment:
This will be problematic right after we start bumping the version. But you might want to rewrite some other code then anyway, so it's likely not a problem.